docs/network.md

```diff
@@ -0,0 +1,16 @@
+# Network ports
+
+Percona Platform requires configuration of network ports to accept monitoring data from PMM. 
+
+Make sure to configure your network and firewall so that all ports listed below are whitelisted and opened for bi-directional communication. UDP is not needed.
```

Comment:
the values in the table are not really ports, this are IP addresses. What is UPD? can we clarify this?